src/app/containers/RadioSchedule/utilities/processRadioSchedule.js

```diff
@@ -0,0 +1,68 @@
+import moment from 'moment';
+import { pathOr, findLastIndex, propSatisfies } from 'ramda/src';
```

Comment:
quick change here.

we should update this import so that we are importing like:
```
import pathOr from 'ramda/src/pathOr';
import findLastIndex from 'ramda/src/findLastIndex';
import propSatisfies from 'ramda/src/propSatisfies';
```